source/packages/modulo_core/src/Cell.cpp

```diff
@@ -327,15 +327,14 @@ const state_representation::CartesianPose Cell::lookup_transform(const std::stri
 
 rclcpp_lifecycle::node_interfaces::LifecycleNodeInterface::CallbackReturn Cell::on_configure(const rclcpp_lifecycle::State&) {
   RCUTILS_LOG_INFO_NAMED(get_name(), "on_configure() is called.");
+  this->active_ = false;
```

Comment:
just as a curiosity, what happens if we call configure() when a node has already been activated?

Reply:
You can't actually. ROS won't let you do it. So in fact this line is problably not needed

Reply:
I think it might be good to remove `this->active_ = false;` in that case